### PR TITLE
[ADD] description to route

### PR DESCRIPTION
**Pull request by mehdignu**
- [ ] Bug fix
  - [X] New feature
  - [ ] BC breaks
  - [ ] Deprecations
  - [ ] Changelog update (if required)

for SEO purposes, descriptions in routes are needed

Fixed tickets : comma separated list of tickets fixed by the PR
License: AGPLv3

### CODE REVIEW

**Comment by ahilles107** [comment]
@mehdignu  -this one is ok, bu tests for bundle are failing. Please fix them. You can run them with:
```
cd src/SWP/Bundle/ContentBundle && composer install --no-interaction --prefer-dist && bin/phpunit
```

File: src/SWP/Bundle/CoreBundle/Migrations/2019/07/Version20190702120702.php [hunk 1]
```diff
@@ -18,6 +18,7 @@ public function up(Schema $schema): void
         $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
 
         $this->addSql('ALTER TABLE swp_user ADD external_id VARCHAR(255) DEFAULT NULL');
+        $this->addSql('ALTER TABLE swp_route ADD COLUMN description VARCHAR(255) DEFAULT NULL');
```
Comment (ahilles107) [inline src/SWP/Bundle/CoreBundle/Migrations/2019/07/Version20190702120702.php]: this is wrong - it should be new migration. As we have already instances with this one marked as done.

**Comment by ahilles107** [comment]
@mehdignu ping

**Comment by ahilles107** [comment]
@mehdignu can You look at those content bundle tests (in most of them description is missing in expected data set)

File: src/SWP/Bundle/ContentBundle/Tests/Functional/Controller/RouteControllerTest.php [hunk 2]
```diff
@@ -408,7 +394,7 @@ public function testFilterRoutesByType()
         $content = json_decode($client->getResponse()->getContent(), true);
         self::assertEquals(
             json_decode(
-                '{"page":1,"limit":10,"pages":1,"total":2,"_links":{"self":{"href":"\/api\/v2\/content\/routes\/?page=1&limit=10"},"first":{"href":"\/api\/v2\/content\/routes\/?page=1&limit=10"},"last":{"href":"\/api\/v2\/content\/routes\/?page=1&limit=10"}},"_embedded":{"_items":[{"id":1,"content":null,"static_prefix":"\/route1","variable_pattern":null,"children":[],"level":0,"template_name":null,"articles_template_name":null,"type":"content","cache_time_in_seconds":1,"name":"route1","description":null,"position":0,"parent":null,"_links":{"self":{"href":"\/api\/v2\/content\/routes\/1"}},"slug":"route1","requirements":[], "lft":1, "rgt": 2},{"id":2,"content":null,"static_prefix":"\/route2","variable_pattern":"\/{slug}","children":[],"level":0,"template_name":null,"articles_template_name":null,"type":"collection","cache_time_in_seconds":2,"name":"route2","description":null,"position":1,"parent":null,"_links":{"self":{"href":"\/api\/v2\/content\/routes\/2"}},"slug":"route2","requirements":{"slug":"[a-zA-Z0-9*\\\-_]+"},"lft":3,"rgt": 4}]}}',
```
Comment (ahilles107) [inline src/SWP/Bundle/ContentBundle/Tests/Functional/Controller/RouteControllerTest.php]: instead removing description from api response - add it to serializer config - `src/SWP/Bundle/ContentBundle/Resources/config/serializer/Model.Route.yml`

**Comment by mehdignu** [comment]
@ahilles107 ping

**Comment by ahilles107** [comment]
@mehdignu do not pick it up in a wrong way, but i see that instead making tests passing with this new description field in them - you just removed it from tests? With this PR we will never know if it still works after some next changes. Please add description fields to tests (route create/update) and check if it's ok in output.

**Pull request closed without merging**